README.md

```diff
@@ -79,18 +81,18 @@ send_timeout                600;
 
 ### Local
 
-To start de the debug its needed the following extensions on the local machine:
+Install the following extensions in the local machine:
 
-- PHP Debug, on VsCode.
-- Xdebug helper, on web browser.
+- PHP Debug, in VsCode.
+- Xdebug helper, in the web browser.
```

Comment:
"on" is a better eng there d: @CarlosAMS123

Reply:
@VicLimaAlamo excelent, on it